src/components/CustomTable/Cells/EditButtonCell/EditButtonCell.js

```diff
@@ -0,0 +1,30 @@
+import EditButton from "@/components/EditButton/EditButton"
+import { TableCell, withStyles } from "@material-ui/core"
+
+const NewTableCell = withStyles((theme) => ({
+  root: {
+    padding: "10px",
+  },
+  head: {
+    backgroundColor: theme.palette.primary.light,
+    color: theme.palette.secondary.dark,
+    fontWeight: theme.typography.fontWeightBold,
+  },
+  body: {
+    borderBottomColor: theme.palette.primary.dark,
+    textTransform: "capitalize",
```

Comment:
Why do you have `textTransform` in a cell that doesn't render text? I also see `fontWeight` being used above,  please make sure your styling makes sense for the components you are styling